include/odp/api/spec/ipsec.h

```diff
@@ -1164,6 +1324,30 @@ void odp_ipsec_config_init(odp_ipsec_config_t *config);
  */
 int odp_ipsec_config(const odp_ipsec_config_t *config);
 
+/**
+ * Inline IPsec config for a particular pktio.
+ *
+ * Ínitialize and configure IPsec for use in inline processing mode with
+ * a given pktio. This must be called after odp_ipsec_config() has been called
+ * with ODP_IPSEC_OP_MODE_INLINE inbound_mode or outbound_mode, after the
+ * pktio has been configured and before any inline SAs is created with
+ * a scope that includes the pktio.
+ *
+ * Calling this function multiple times for the same open pktio results in
+ * undefined behaviour.
+ *
+ * @param pktio    Inline-enabled open pktio for which the configuration is
+ *                 done
+ * @param config   Pointer to IPSEC configuration structure
+ *
+ * @retval 0 on success
+ * @retval <0 on failure
+ *
+ * @see odp_ipsec_inline_capability(), odp_ipsec_config_init()
+ */
+int odp_ipsec_inline_config(odp_pktio_t pktio,
+			    const odp_ipsec_config_t *config);
```

Comment:
Some of the contents in odp_ipsec_config_t such as mode is not probably needed.
Can the required inline related config be part of odp_pktio_config_t  instead of this new API ?